database/db_build.js

```diff
@@ -0,0 +1,15 @@
+// This file should be run once to populate the database with seed data!!
+
+const Career = require('./career_model.js');
```

Comment:
what is this file for ??

Reply:
I see this file only being used in development not production.
so it's more of a development tool to populate the database.
best solution is to .gitignore it or if you have something else please comment

Reply:
I agree :+1: